collections/_article/release-candidate-godot-4-2-rc-2.md

```diff
@@ -0,0 +1,101 @@
+- As a reminder, since beta 5, we provide pre-compiled ARM64 and ARM32 Linux binaries (editor and templates). This RC 2 improves this new set of binaries by temporarily disabling <abbr title="Link Time Optimization">LTO</abbr>, which seemed to cause crashes on at least Raspberry Pi OS. Please try it out if you have such ARM Linux systems!
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also [try the **Web editor**](https://editor.godotengine.org/releases/4.2.rc2/) or the **Android editor** for this release. If you are interested in the latter, please request to join [our testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds.
+
+-----
+
+*The illustration picture is from a pixel art [2D platformer prototype](https://github.com/securas/2023_GithubGameOff) by [Securas](https://twitter.com/Securas2010), made with Godot 4.2 RC 1. It was originally developed for the ongoing [GitHub Game Off](https://itch.io/jam/game-off-2023) game jam (which is very popular among Godot users, and which [Securas won in 2019](https://github.blog/2020-01-14-game-off-2019-winners/#first-place-sealed-bite)), but due to circumstances Securas had to cut it short this time and [open sourced the prototype](https://github.com/securas/2023_GithubGameOff) under MIT (code) and CC0 (assets) licenses. It's a great resource for platformer gameplay, pixel art graphics and Securas' now famous 2D water reflection shaders!*
+
+## What's new
+
+For an overview of what's new overall in Godot 4.2, have a look at the release notes for [4.2 beta 1](/article/dev-snapshot-godot-4-2-beta-1/), which cover a lot of the changes. This blog post only covers the changes between RC 1 and RC 2.
+
+**13 contributors** submitted **34 improvements** for this release. You can review the complete list of changes with our [interactive changelog](https://godotengine.github.io/godot-interactive-changelog/#4.2-rc2), which contains links to relevant commits and PRs for this and every previous release. Below are the most notable changes compared to 4.2-rc1:
+
+- Animation: Fix seeking bug in AnimationPlayerEditor ([GH-85193](https://github.com/godotengine/godot/pull/85193)).
+- Animation: Clear seeked/started flag after seeking/advancing in AnimationPlayer ([GH-85221](https://github.com/godotengine/godot/pull/85221)).
+- Animation: Bind `_reset`/`_restore` in AnimationMixer ([GH-85254](https://github.com/godotengine/godot/pull/85254)).
+- Animation: Fix TrackCache memory crash ([GH-85266](https://github.com/godotengine/godot/pull/85266)).
+- Animation: Perform safe copies in `AnimatedValuesBackup::get_cache_copy()` ([GH-85302](https://github.com/godotengine/godot/pull/85302)).
+- Animation: Fix a crash when trying to restore uncopyable animation tracks ([GH-85308](https://github.com/godotengine/godot/pull/85308)).
```

Comment:
I would normally add such lines to the line of the original PR without listing it explicitly, since it's a part of the same package.